conans/client/rest/rest_client_common.py

```diff
@@ -156,7 +156,11 @@ def get_json(self, url, data=None):
             response.charset = "utf-8"  # To be able to access ret.text (ret.content are bytes)
             raise get_exception_from_error(response.status_code)(response.text)
 
-        result = json.loads(decode_text(response.content))
+        try:  # This can fail, if some proxy returns 200 and an html message
+            result = json.loads(decode_text(response.content))
+        except Exception:
```

Comment:
The issue is produced because of a message that it is not a json, right? The right way to check this is to check if `application/json` header is present. I know you would think that is risky, but a server returning a json without a json header is a bug. The protocols need to be followed. If our own test's with the server fails (I hope they don't) we need to fix it in the server but I agree we cannot break old servers, in that case, narrow the exception capturing to the json decoding, please.

Reply:
We need also to test with Artifactory.

Reply:
Done.
Tested manually with Artifactory CE 6.3 and Bintray